### PR TITLE
Update README

### LINKED ISSUE #48: Update README.md
The readme file is a bit out of date, we know have a really good intro that landed in #47 and it might be worth to link to this instead and leave README.md for a more technical documentation of the repository (building, running, organisation, etc).

### PR DESCRIPTION
**Pull request by adoerr**
Resolves #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,80 @@
 # BEEFY
 **BEEFY** (**B**ridge **E**fficiency **E**nabling **F**inality **Y**ielder) is a secondary
 protocol running along Grandpa Finality to support efficient bridging with non-Substrate
```
Comment (tomusdrw) [inline README.md:3]: By occasion let's maybe change all `Grandpa` occurrences to `GRANDPA`?

File: README.md [hunk 2]
```diff
@@ -1,105 +1,80 @@
...
+[Getting Started](https://substrate.dev/docs/en/knowledgebase/getting-started/) page is a good
+resource.
+
+## Documentation
+
+The best way to get going with BEEFY is by reading the [walkthrough](./docs/walkthrough.md) document!
+This document puts BEEFY into context and provides motivation for why this project has been started.
+In addition to that the current status as well as a preliminary roadmap is presented.
+
+[beefy](./docs/beefy.md) is a collection of early notes and ideas, stil worth checking out.
```
Comment (tomusdrw) [inline README.md:46]:
```suggestion
[BEEFY brainstorming](./docs/beefy.md) is a collection of early notes and ideas, stil worth checking out.
```

File: README.md [hunk 3]
```diff
@@ -1,105 +1,80 @@
...
+
+## Project Layout
+
+What follows is an overview of how the project repository is laid out. The main components are the
+`beefy-gadget` which is a POC of the BEEFY round logic. The BEEFY `pallet` which is mainly a thin
```
Comment (tomusdrw) [inline README.md:51]:
```suggestion
`beefy-gadget` which is a POC of the BEEFY round logic. The BEEFY `pallet` which is mainly a thin
```
Might be worth either renaming the pallet folder to `pallet-beefy` maybe or `beefy-gadget` folder to simply `gadget/service`?

File: README.md [hunk 4]
```diff
@@ -1,105 +1,80 @@
...
+
+## Running BEEFY
+
+Currently the easiest way to see BEEFY in action is to run a single dev node like so:
+
+```
+$ RUST_LOG=beefy=trace ./target/debug/node-template --tmp --dev --alice --validator
+```
+
+Expect additional (more usefull) deployment options to be added soon.
```
Comment (tomusdrw) [inline README.md:80]:
```suggestion
Expect additional (more useful) deployment options to be added soon.
```

**Review by tomusdrw — COMMENTED** [review]
overall looks good, few minor grumbles.

File: README.md (hunk 2 above)
Comment (HCastano) [inline README.md]:
```suggestion
[beefy](./docs/beefy.md) is a collection of early notes and ideas, still worth checking out.
```

File: README.md [hunk 5]
```diff
@@ -1,105 +1,80 @@
...
+
+## Project Layout
+
+What follows is an overview of how the project repository is laid out. The main components are the
+`beefy-gadget` which is a POC of the BEEFY round logic. The BEEFY `pallet` which is mainly a thin
+integration layer over the session pallet and keeps track of the current authorities.
+Finally the BEEFY `primitives` crate which contains most of the type definitions for the 
+BEEFY protocol.
+
+The `primitives` crate also contains a test [light_client](.primitives/tests/light_client/) which demonstarates how BEEFY would
```
Comment (HCastano) [inline README.md:56]:
```suggestion
The `primitives` crate also contains a test [light_client](.primitives/tests/light_client/) which demonstrates how BEEFY would
```

**Review by HCastano — APPROVED** [review]
Looks good as long as Tomek's comments are addressed

File: README.md (hunk 3 above)
Comment (adoerr) [reply README.md:51]: I will go with `beefy-pallet`. I would like to rework/rename `node-example` to `beefy-node` in a later PR as well.

**Pull request merged**